modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/contract/DayOfWeekJobId.java

```diff
@@ -0,0 +1,30 @@
+package org.motechproject.scheduler.contract;
+
+import org.motechproject.event.MotechEvent;
+
+/**
+ * Jobs ID for {@link DayOfWeekSchedulableJob}s.
+ */
+public class DayOfWeekJobId extends JobId {
```

Comment:
By the way, this may be a good time to start adding suffix to cron expressions as well. Isn't it "null" for cron expressions now?